src/registries/base-registry.js

```diff
@@ -171,7 +171,9 @@ export default class BaseRegistry {
       key = key.replace(/__/g, '.');
 
       // replace underscores with dashes
-      key = key.replace(/_/g, '-');
```

Comment:
Can you instead add it in a more generic way that supports `__`?

For example: `key.replace(/([^_])_/g, '$1-')`

Reply:
That's a good idea. We're probably assuming any key starting with an underscore needs to stay an underscore, but that's probably always true

Reply:
@arcanis Are the changes satisfactory?